project/Common.scala

```diff
@@ -51,25 +51,27 @@ object Common extends AutoPlugin {
         else
           scalacOptionsBase
       },
-      javacOptions ++= Seq(
+      Compile / javacOptions ++= Seq(
           "-Xlint:unchecked"
         ),
-      javacOptions ++= (
+      Compile / javacOptions ++= (
           if (isJdk8) Seq.empty
           else Seq("--release", "8")
         ),
       scalacOptions ++= (
           if (isJdk8 || scalaVersion.value == Dependencies.Scala212) Seq.empty
           else Seq("--release", "8")
         ),
+      Compile / doc / javacOptions := Nil,
```

Comment:
Not sure why this one was failing as well, but it was on my machine (javadoc on JDK 17 issue?). Anyway, no need for Xlint during javadoc generation.